plugins/woocommerce-admin/client/marketing/overview-multichannel/DiscoverTools/DiscoverTools.tsx

```diff
@@ -133,8 +133,75 @@ export const DiscoverTools = () => {
 
 					return (
 						<>
-							<CardDivider />
-							{ renderPluginCardBodies( subcategoryPlugins ) }
+							{ subcategoryPlugins.map( ( el ) => {
+								return (
+									<Fragment key={ el.product }>
```

Comment:
Thanks for moving this down here!  Using any function prefixed with `render` always feels like a code smell.

Optional, but this might even be worth moving to its own component if we're already moving this code.

Reply:
> Optional, but this might even be worth moving to its own component if we're already moving this code.

I actually did try that out with a new component, but by doing that, the component would then have to expose props like `onInstallAndActivate` and we have to pass things down from the parent component to the child component. The component and its API felt like not making sense to me, so I kept it this way with a flatter approach without another component level.

Reply:
Passing props down to a component is not necessarily a bad thing.  If you find yourself passing a lot of props, it may represent an underlying structural issue with the components.

That said, don't feel like this is a blocker for this PR.  The way it's done here is not bad.